src/qutip_qip/circuit_latex.py

```diff
@@ -227,9 +228,17 @@ def image_from_latex(code, file_type="png"):
             try:
                 os.chdir(temporary_dir)
                 with open(filename + ".tex", "w") as file:
-                    file.write(_latex_template % (_qcircuit_latex_min, code))
-                _run_command((_pdflatex, '-interaction', 'batchmode',
-                              filename))
+                    file.write(_latex_template % code)
+                try:
+                    _run_command((_pdflatex, '-interaction', 'batchmode',
+                                  filename))
+                except RuntimeError as e:
+                    message = (
+                        "pdflatex failed."
+                        " Perhaps you do not have it installed, or you are"
+                        " missing the LaTeX package 'qcircuit'."
```

Comment:
@jakelishman Thanks for this ! `qcircuit` not installed was one of the issues I was facing in #60.

Reply:
Just having a flick through #60, it looks like you might be looking for too much on `conda` - `imagemagick` is something you'd typically expect to find in a system package manager, rather than `conda`.  On Debian-like Linuxes that would be likely be `apt` (`apt[-get] install imagemagick`), or `brew install imagemagick` on Mac.

`qcircuit` itself should be present in any fully featured TeXLive installation.  If you've just got a minimal TeXLive it might not have it, but you can use `tlmgr` to install it.

Reply:
> qcircuit itself should be present in any fully featured TeXLive installation

I had a minimal install with `qcircuit` but not all of it's dependencies were installed. I used [this](https://pypi.org/project/latextools/). 

> imagemagick is something you'd typically expect to find in a system package manager

How would you typically know if some dependency could be found in a package manager instead of `conda` ?

Reply:
That `latextools` package seems to be more about building and manipulating LaTeX source in Python, rather than actually providing the LaTeX distribution - it still needs TeXLive or something.

Conda often has more than I expect it to, but generally command-line tools unrelated to Python would be installed from the system package manager (because they're system packages). Conda does blur the lines a little bit (especially when it comes to BLAS), but `imagemagick` and `pdflatex` would normally be system commands.

Reply:
TeXLive comes with `tlmgr` (its own package manager for LaTeX files), so a minimal installation for your purposes here would probably be
```bash
$ apt-get install imagemagick pdfcrop pdf2svg
$ apt-get install texlive  # fairly minimal version, but I think it has amsmath and amsfonts
$ tlmgr install braket qcircuit
```
(granted I'm guessing a little, but that's the gist)

Reply:
Thanks !  I will test this in a new virtual environment when I update the documentation. 

I used `texlive` as well but I had to go on a search for a couple of more `latex` packages based on the errors.